<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Nizikano</class>
 <widget class="QWidget" name="Nizikano">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nizikano</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="4">
    <widget class="QGroupBox" name="gbTicker">
     <property name="title">
      <string>Notification</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lNotification">
        <property name="text">
         <string>...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="AspectWebView" name="webview" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pbBack">
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="wFooter" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pbGacha">
        <property name="text">
         <string>Gacha</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbShops">
        <property name="text">
         <string>Shops</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbEvents">
        <property name="text">
         <string>Events</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbHome">
        <property name="text">
         <string>Home</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbMenu">
        <property name="text">
         <string>Menu</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QWidget" name="" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pbTop">
        <property name="text">
         <string>Top</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbID">
        <property name="text">
         <string>Change ID</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="cbRefresh">
        <property name="text">
         <string>AutoRefresh:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbTimeout">
        <property name="suffix">
         <string> min</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>120</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbItem">
        <property name="text">
         <string>Items</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbGroup">
        <property name="text">
         <string>Groups</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbDegree">
        <property name="text">
         <string>Achievements</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbAlbum">
        <property name="text">
         <string>Albums</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbFriend">
        <property name="text">
         <string>Friends</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbRandom">
        <property name="text">
         <string>Random Visit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbBook">
        <property name="text">
         <string>Enstylopedia</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbCampain">
        <property name="text">
         <string>Information</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbStatus">
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbOption">
        <property name="text">
         <string>Other</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbHelp">
        <property name="text">
         <string>Help</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lDebug">
        <property name="text">
         <string>Debug:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLineEdit" name="leURL"/>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pbGo">
     <property name="text">
      <string>GO!</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AspectWebView</class>
   <extends>QWidget</extends>
   <header>aspectwebview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
